source/NoteWindow.cpp

```diff
@@ -77,38 +77,27 @@ const struct tm gettime() {
 }
 
 // Constructor
-NoteWindow::NoteWindow(int32 id)
-		  : BWindow (BRect(100,100,350,350) , "TakeNotes", B_DOCUMENT_WINDOW, B_ASYNCHRONOUS_CONTROLS){
-
-	BString 	title("Untitled Note ");
+NoteWindow::NoteWindow()
+		  : BWindow (BRect(100,100,350,350) , "Untitled", B_DOCUMENT_WINDOW, B_ASYNCHRONOUS_CONTROLS), fRef(NULL){
 
 	// Init the windows to create the menu
 	InitWindow();
 
 	_CreateNoteView();
 
-	// Set the title
-	title << id;
-	SetTitle(title.String());
-
 	// It will be associated to the window
 	AddChild(fNoteView);
 
-	// Creating the file panels
-	// Save file panel
+	// Create the save file panel
 	fSavePanel = new BFilePanel(B_SAVE_PANEL, new BMessenger(this), NULL, 0,
 		false);
 
-	// Open file panel
-	fOpenPanel = new BFilePanel (B_OPEN_PANEL, new BMessenger (this), NULL, B_FILE_NODE, false, NULL,
-		&fNoteRefFilter);
-
 	Show();
 }
 
 // Construtor
 NoteWindow :: NoteWindow(entry_ref *ref)
-	: BWindow (BRect(100,100,350,350) , "TakeNotes", B_DOCUMENT_WINDOW, B_ASYNCHRONOUS_CONTROLS){
+	: BWindow (BRect(100,100,350,350) , "TakeNotes", B_DOCUMENT_WINDOW, B_ASYNCHRONOUS_CONTROLS), fRef(ref){
```

Comment:
Same coding style change as the above default constructor. See also: [Haiku Coding Guidelines](https://www.haiku-os.org/development/coding-guidelines/)